apps/api/src/app/integrations/e2e/create-integration.e2e.ts

```diff
@@ -469,7 +469,7 @@ describe('Create Integration - /integration (POST)', function () {
       body: { data },
     } = await session.testAgent.post('/v1/integrations').send(payload);
 
-    expect(data.priority).to.equal(2);
+    expect(data.priority).to.equal(1);
```

Comment:
Now that the first integration created in setup is `primary`, the next created integration will have a higher `priority` per existing logic.